docs/documentation/development/bindings/categories.md

```diff
@@ -14,7 +14,44 @@ The thing type definition allows to specify a category. User interfaces can pars
 
 | Category        | Description                                          |
 |-----------------|------------------------------------------------------|
-
+| Aircondition    | Air condition devices incl. Fans |
+| BinarySensor    | General purpose sensor that returns a boolean value, i.e. a contact that reports open/close on a door or window |
+| Bridge          | Bridges/Gateway need to access other devices like used by Philips Hue for example |
+| Boiler          | Devices that heat up water |
+| Camera          | All kinds of cameras |
+| Car             | Smart Cars |
+| Cistern         | Cistern |
+| Clock           | Devices that provide the current time |
+| Coffee          | Coffee machines |
+| Fridge          | Fridges, Freezers |
+| Furniture       | Smart furniture |
+| GarageDoor      | Garage doors |
+| GeneralSensor   | General purpose sensor that does not fit in any category |
+| Heating         | Devices that deal with heating |
```

Comment:
too broad / too functional. Maybe "Radiator"? But they are usually not connected anyhow. What we often have are ["RadiatorControls"](https://www.google.de/search?q=Radiator+Controls&tbm=isch).